src/views/list/content.php

```diff
@@ -16,6 +16,8 @@
 
 <div id="tribe-events-content" class="tribe-events-list">
 
+	<?php do_action('tribe_events_list_before_the_content'); ?>
```

Comment:
Looks good! Be even sweeter if we could add a docblock in here though (and a dab of whitespace before and after the single quotes) :bow: